get_started.md

```diff
@@ -2,42 +2,59 @@
 
 The Gaia Sandbox provides a containerized simulation environment that allows you to experiment with samples written using the Gaia Platform.
 
-This sample models a factory that contains the following:
+This sample models a security access control to allow or deny people entering an office campus. It contains:
 
-TODO
+- A headquarters (HQ) building
+- Rooms inside the HQ building
+- People, of varying security access levels
+    - John: an employee with access to all rooms
+    - Jane: a visitor with limited room access
+    - A stranger: they are allowed inside the HQ building!
+- Event schedules for people and rooms
 
 ## Overview of the Sandbox
 
 The Sandbox UI consists of the following elements.
 
-### Simulator window
+### User interface window
 
-TODO
+The right side of the screen shows the controls for the Access Control simulation. Each person in the **People** column has several icons: **Badge**, **Face**, **Parking**, and **WiFi**. These icons are all scanning options: for example, clicking on somebody's **Badge** icon simulates a "badge scan" into a building.
+
+Every person has a drop-down menu of the buildings/rooms they can scan into. Click on that box to choose a building/room, then click on one of the scan icons to choose a way to scan into that building/room. Every drop-down menu starts with only the **HQ Building** because you must scan into a building before you can scan into a room.
+
+You can also control time! Click the fast-forward icon by **Location at time** to set the time, which is important for visitors like Jane who cannot enter a room until a certain time.
```

Comment:
Avoid the use of ! it really isn't necessary.

The  **Location at time** control allows you to set the time of day, which is important for granting access to visitors such as Jane who cannot enter a room until a certain time.  The fast-forward icon advances the time in 30-minute intervals.